lib/compiler/graph-compiler.js

```diff
@@ -462,7 +462,7 @@ var GraphCompiler = CodeGenerator.extend({
     gen_Sink: function(ast) {
         // silly special case for ast.name due to sink's special-case parsing
         // and AST.
-        var proc = this.gen_proc(ast, 'clientsink', '{name: "'+ast.name+'"}');
+        var proc = this.gen_proc(ast, 'clientview', '{name: "'+ast.name+'"}');
```

Comment:
Although I think this should simply be 'view' and not 'clientview', that can be done in a subsequent change when we also rename the AST nodes from `Sink` to `View`. At that point I think we can just use 'view' consistently.